sys-cluster/charliecloud/charliecloud-0.39.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit autotools optfeature python-single-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/${PN}/main.git"
+else
+	SRC_URI="https://gitlab.com/${PN}/main/-/archive/v${PV}/main-v${PV}.tar.bz2 -> ${P}.tar.bz2"
+	KEYWORDS="~amd64 ~x86 ~x86-linux"
+	S="${WORKDIR}/main-v${PV}"
+fi
+
+DESCRIPTION="Lightweight user-defined software stacks for high-performance computing"
+HOMEPAGE="https://hpc.github.io/charliecloud/"
+LICENSE="Apache-2.0"
+
+SLOT="0"
+IUSE="ch-image doc"
+
+# Extensive test suite exists, but downloads container images
+# directly and via Docker and installs packages inside using apt/yum.
+# Additionally, clashes with portage namespacing and sandbox.
+RESTRICT="test"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="${PYTHON_DEPS}
```

Comment:
Are you sure that `PYTHON_DEPS` in `RDEPEND` is enough? There is amazing Gentoo python guide describing most of use cases, see https://projects.gentoo.org/python/guide/single.html

Reply:
Indeed, while Python is not called during the build process (only indirectly via sphinx), there is version detection and other logic in the Makefile, so it should be in `BDEPEND` too. Thanks for the link to the great documentation, I was unaware of this helpful resource!